src/http-api.ts

```diff
@@ -5,25 +5,35 @@ import responseTime from 'response-time';
 
 import cors from 'cors';
 import express from 'express';
-import { register, Histogram } from 'prom-client';
+import { register, Histogram, Counter } from 'prom-client';
+import morgan from 'morgan';
 
 import { v4 as uuidv4 } from 'uuid';
 
 import { jsonRpcApi } from './json-rpc-api.js';
+import { JsonRpcRequest } from './lib/json-rpc.js';
 import { createJsonRpcErrorResponse, JsonRpcErrorCodes } from './lib/json-rpc.js';
 
 export class HttpApi {
+  requestCounter = new Counter({
+    name       : 'dwn_requests_total',
+    help       : 'all dwn requests processed',
+    labelNames : ['method', 'status', 'error'],
```

Comment:
I'm sure there are other things we could record about all requests, these seemed like a couple of easy candidates.